docs/badge.md

```diff
@@ -142,3 +172,72 @@ Custom component to replace the badge outer component
 |          Type          |                  Default                   |
 | :--------------------: | :----------------------------------------: |
 | React Native Component | View, if `onPress` then `TouchableOpacity` |
+
+---
+
+## Reference for withBadge
+
+### `value`
+Text value to be displayed by badge, defaults to empty
+
+|                    Type                    | Default |
+| :----------------------------------------: | :-----: |
+| String OR Number OR React Native Component |  none   |
```

Comment:
Oops I just forgot this one: `value` can also be a callback that takes the props as parameter

Reply:
Added it.